aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginTest.java

```diff
@@ -170,19 +178,49 @@ public void clearStopsSyncUntilNextInteraction() throws AmplifyException, JSONEx
 
         assertRemoteSubscriptionsStarted();
 
-        Person person1 = createPerson("Test", "Dummy I");
-        Person person2 = createPerson("Test", "Dummy II");
-
-        synchronousDataStore.save(person1);
-        Person result1 = synchronousDataStore.get(Person.class, person1.getId());
-        assertEquals(person1, result1);
+        BlogOwner blogOwner1 = createBlogOwner("Test", "Dummy I");
+        BlogOwner blogOwner2 = createBlogOwner("Test", "Dummy II");
+
+        when(mockApiPlugin.mutate(any(), any(), any())).thenAnswer(invocation -> {
+            String data = new JSONObject()
+                .put("id", blogOwner1.getId())
+                .put("name", blogOwner1.getName())
+                .put("_deleted", false)
+                .put("_version", 1)
+                .put("_lastSyncedAt", Time.now())
+                .toString();
+            GraphQLResponse<String> response = new GraphQLResponse<>(data, Collections.emptyList());
+            ((Consumer<GraphQLResponse<String>>) invocation.getArgument(1)).accept(response);
+            return /* void */ null;
+        });
+        HubAccumulator blogOwner1Accumulator =
+            HubAccumulator.create(HubChannel.DATASTORE, publicationOf(blogOwner1), 1).start();
+        synchronousDataStore.save(blogOwner1);
+        BlogOwner result1 = synchronousDataStore.get(BlogOwner.class, blogOwner1.getId());
+        assertEquals(blogOwner1, result1);
+        blogOwner1Accumulator.await();
```

Comment:
After you save to local store, need to wait for sync engine to have it (or else your verify() command won't necessarily have 2 invocations, at the end of the test.)